charts/sonarqube/templates/init-sysctl.yaml

```diff
@@ -5,29 +5,43 @@ metadata:
   labels: {{- include "sonarqube.labels" . | nindent 4 }}
 data:
   init_sysctl.sh: |-
+    {{- if .Values.initSysctl.enabled }}
```

Comment:
Considering that all the mounts in the pods are already conditioned on `.Values.initSysctl.enabled,` we could put the if here around the whole resource definition, not creating the config map if it is not needed.

Reply:
I will agree with that. However, this also means the `init-fs.yaml` needs to be changed because the design needs to be persisted.
https://github.com/SonarSource/helm-chart-sonarqube/blob/4951d66d8fdb35d84082a9a44b41dcd3d0e30224/charts/sonarqube/templates/init-fs.yaml

I am committing relative changes for now (including changelog), and please let me know if you disagree about this, so I will revert it.